### PR TITLE
58 add function to create listing

### PR DESCRIPTION
**Pull request by OscarThorburn**

### CODE REVIEW

File: src/GT.UI/Controllers/ListingsController.cs [hunk 1]
```diff
@@ -24,21 +26,23 @@ public ListingsController(IGTListingService listingService, UserManager<Applicat
 
 		// GET: api/Listings
 		[HttpGet]
-		public async Task<ActionResult<IEnumerable<ListingOverviewDTO>>> GetListings(IListingFilterModel filterModel = null)
+		public async Task<ActionResult<IEnumerable<ListingOverviewDTO>>> GetListings(ListingFilterModel filterModel)
 		{
 			// TODO Populate and create a filtermodel
```
Comment (RasmusBroborg) [inline src/GT.UI/Controllers/ListingsController.cs:31]: Should remove outdated TODO

File: src/GT.UI/Controllers/ListingsController.cs [hunk 2]
```diff
@@ -24,21 +26,23 @@ public ListingsController(IGTListingService listingService, UserManager<Applicat
...
 			}
 
 			return Ok(listingDTOs);
-		}
+}
+
 
-		// GET: api/Listings/5
+// GET: api/Listings/5
+
```
Comment (RasmusBroborg) [inline src/GT.UI/Controllers/ListingsController.cs:45]: Remove unnecessary linebreak.

**Review by RasmusBroborg — CHANGES_REQUESTED** [review]

File: src/GT.UI/Controllers/ListingsController.cs (hunk 1 above)
Comment (OscarThorburn) [reply src/GT.UI/Controllers/ListingsController.cs]: Fixed

File: src/GT.UI/Controllers/ListingsController.cs (hunk 2 above)
Comment (OscarThorburn) [reply src/GT.UI/Controllers/ListingsController.cs]: Fixed

**Review by RasmusBroborg — APPROVED** [review]

**Pull request merged**